GenderPayGap.WebUI/Views/AdminOrganisationSector/ChangeSector.cshtml

```diff
@@ -0,0 +1,168 @@
+﻿@using GenderPayGap.Core
+@using GenderPayGap.WebUI.Models.Admin
+@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@model GenderPayGap.WebUI.Models.Admin.AdminChangeSectorViewModel
+
+@{
+    ViewBag.Title = "Change sector - Administration - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+@section BeforeMain {
+    @{
+        var crumbs = new List<CrumbViewModel>
+        {
+            new CrumbViewModel
+            {
+                Text = "Admin",
+                Href = Url.Action("Home", "Admin")
+            },
+            new CrumbViewModel
+            {
+                Text = Model.Organisation.OrganisationName,
+                Href = Url.Action("ViewOrganisation", "AdminViewOrganisation", new {id = Model.Organisation.OrganisationId})
+            },
+            new CrumbViewModel
+            {
+                Text = "Sector history",
+                Href = Url.Action("ViewSectorHistory", "AdminOrganisationSector", new {id = Model.Organisation.OrganisationId})
+            },
+            new CrumbViewModel
+            {
+                Text = "Change sector"
+            }
+        };
+    }
+
+    @(Html.GovUkBreadcrumbs(new BreadcrumbsViewModel
+    {
+        Crumbs = crumbs
+    }))
+}
+
+<span class="govuk-caption-xl">Administration</span>
+<h1 class="govuk-heading-xl">
+    Change sector
+    <br/>
+    <span class="govuk-!-font-size-27">
+        for @(Model.Organisation.OrganisationName)
+    </span>
+</h1>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <form method="POST" action="@Url.Action("ChangeSectorPost", "AdminOrganisationSector", new {id = Model.Organisation.OrganisationId})">
+            @(Html.AntiForgeryToken())
+
+            @(Html.Hidden(nameof(AdminChangeSectorViewModel.Action),
+                ChangeOrganisationSectorViewModelActions.OfferNewSectorAndReason))
+
+            @Html.GovUkErrorSummary()

```

Comment:
**(Minor, but useful to know)**

When I tested this, the error messages appear the "wrong" way round.

The fields are in the order (sector, reason) but the errors appear in the order (reason, sector).
The `@Html.GovUkErrorSummary()` has a way of specifying the order of the error messages:
```
@(Html.GovUkErrorSummary(new[]
{
    nameof(AdminChangeSectorViewModel.NewSector),
    nameof(AdminChangeSectorViewModel.Reason)
}))
```